tex/gregoriotex-spaces.tex

```diff
@@ -29,38 +29,51 @@
 %% The following macros enable users to tune penalties used in Gregorio
 
 % macro to force a break on a new line
-\def\GreForceBreak{\gre@penalty{\grenewlinepenalty}}%
+\def\GreForceBreak{\gre@penalty{\the\gre@space@count@newlinepenalty}}%
 
 % macro to prevent a line break
-\def\GreNoBreak{\gre@penalty{\grenobreakpenalty}}%
+\def\GreNoBreak{\gre@penalty{\the\gre@space@count@nobreakpenalty}}%
 
 
+
+\newcount\gre@hyphenpenaltysave%
+\newcount\gre@exhyphenpenaltysave%
+\newcount\gre@doublehyphendemeritssave%
+\newcount\gre@finalhyphendemeritssave%
+\newcount\gre@brokenpenaltysave%
+\newcount\gre@loosenesssave%
+\newcount\gre@tolerancesave%
+\newcount\gre@pretolerancesave%
+\newskip\gre@emergencystretchsave%
+\newcount\gre@widowpenaltysave%
+\newcount\gre@clubpenaltysave%
```

Comment:
This is the second time you've declared some of these save counts (they are also declared in gregoriotex-main.tex).  You should only need to declare them once.